src/scene/data.ts

```diff
@@ -44,6 +47,61 @@ export function saveAsJSON(
   );
 }
 
+export async function exportToShortlink(
+  elements: readonly ExcalidrawElement[]
+) {
+  const serialized = JSON.stringify({
+    version: 1,
+    source: window.location.origin,
+    elements: elements.map(({ shape, ...el }) => el)
+  });
+
+  // Generating a random unique shortcode
+  // https://stackoverflow.com/questions/6248666/
+  const firstPart = (Math.random() * 46656) | 0;
+  const secondPart = (Math.random() * 46656) | 0;
+  const shortcode =
+    ("000" + firstPart.toString(36)).slice(-3) +
+    ("000" + secondPart.toString(36)).slice(-3);
+
+  const response = await fetch(
+    JSONSTOREIO_SHORTLINK_ENDPOINT + "/" + shortcode,
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: serialized
+    }
+  );
+  const json = await response.json();
+  if (json.ok) {
+    const url = new URL(window.location.href);
+    url.searchParams.append("share", shortcode);
+
+    await navigator.clipboard.writeText(url.toString());
+    window.alert("Copied shareable link " + url.toString() + " to clipboard");
+  } else {
+    window.alert("Couldn't create shareable link");
+  }
+}
+
+export async function importFromShortlink(shortcode: string | null) {
+  let elements: readonly ExcalidrawElement[] = [];
+  const response = await fetch(
+    JSONSTOREIO_SHORTLINK_ENDPOINT + "/" + shortcode
+  ).then(data => data.text());
+  if (response != null) {
+    try {
+      const json = JSON.parse(response);
+      elements = json.result.elements || [];
+    } catch (e) {
+      // Show an error message that importing failed
```

Comment:
Can you do something like doing an alert and doing console.error for now? So that it doesn’t silently fail